azure/packages/azure-client/package.json

```diff
@@ -14,23 +14,26 @@
 	"exports": {
 		".": {
 			"import": {
-				"types": "./lib/index.d.ts",
+				"types": "./lib/azure-client-untrimmed.d.ts",
```

Comment:
It would probably be worth sanity checking the intellisense / VSCode navigation behaviors with this change. For our untrimmed types, it may actually be better to just use the unmodified `index.d.ts` files instead othe API-Extractor output, as the source map behaviors might be a bit better.

Reply:
Do you know what to look for in particular? I'm not sure how to test this.

Reply:
The reason I did this change, though, was so that something would fail if the rollup wasn't produced before a downstream
package builds.

Reply:
Well, not this package, but the others.

Reply:
Makes sense. I would probably test it by building the packages, then reload vscode as a sanity measure, then trying to ctrl+click through definitions across package boundaries. Under normal circumstances, this should navigate to the source code containing the definition but may end up navigating to the generated `d.ts` module instead. It's probably not the end of the world if it does this (it definitely will for any of our special import paths regardless), but it would be nice to know so we can communicate expectations.

Reply:
I just reverted that part of the change for now. We can enable the api-extractor builds then separately test/change the default types. There's a risk that we'll have a missing dependency or something that we won't catch until we make those changes, but we'll catch it then.